conf/app.src

```diff
@@ -1,5 +1,5 @@
-SOURCE_URL=https://git.tt-rss.org/git/tt-rss/archive/153cb6d30510f9804fe596ac6777bc2f35111ae2.tar.gz
-SOURCE_SUM=1413d5d2ce6e4dfd63f4c69bb79f6e18287313756329eaf61928ce25085ebc13
+SOURCE_URL=https://git.tt-rss.org/git/tt-rss/archive/925065b1fed03710cef9018c2078b95b0fa425ad.tar.gz
```

Comment:
Why not using https://git.tt-rss.org/fox/tt-rss/archive/19.2.tar.gz? You can find all releases here: https://git.tt-rss.org/fox/tt-rss/releases

Reply:
Good catch! The development had historically switched out of formal releases, hence we had switched to commit references... which isn't the case any more today.